packages/vscode-mdx/src/typescript-plugin.js

```diff
@@ -0,0 +1,107 @@
+/**
```

Comment:
I think it would be nice to publish this as `@mdx-js/typescript-plugin`, but we can do that later.